src/TR_alg.jl

```diff
@@ -191,11 +201,11 @@ function TR(
 
     if η2 ≤ ρk < Inf
       Δk = max(Δk, γ * sNorm)
-      set_radius!(ψ, Δk)
```

Comment:
Why did this disappear? The radius should be updated here (not when eta1 ≤ rhok < eta2).

Reply:
$\eta_1 \le \eta_2$, so it will be updated anyways just a few lines below.

Reply:
I see. I think it's less clear though. When I read it, I have the impression that there is a bug.